runtime/Makefile

```diff
@@ -97,23 +129,25 @@ dummy_device:
 .PHONY: test
 test: $(RT_BUILD_DIR)/tests/runner_tests_lightning
 	@echo "test the Catalyst runtime test suite"
-	$(ASAN_COMMAND) $(RT_BUILD_DIR)/tests/runner_tests_lightning
-
 ifeq ($(ENABLE_OPENQASM), ON)
+	# Test the OpenQasm devices C++ tests
 	$(ASAN_COMMAND) $(RT_BUILD_DIR)/tests/runner_tests_openqasm
 endif
+	# Test the Lightning suite C++ tests
+	$(ASAN_COMMAND) $(RT_BUILD_DIR)/tests/runner_tests_lightning
```

Comment:
Any reason the order was switched? (just curious)

Reply:
Lightning simulators support change and fail more frequently than OpenQasm devices. so, I thought this is good to always check OpenQasm tests at first (there are just a few and usually pass :green_circle:) and then check the C++ unit tests of Lightning devices.

Reply:
happy to revert this change if you think otherwise. (no strong opinion from my side)

Reply:
no worries, fine by me